core/math/quat.cpp

```diff
@@ -135,11 +138,17 @@ bool Quat::is_normalized() const {
 }
 
 Quat Quat::inverse() const {
+#ifdef MATH_CHECKS
+	ERR_FAIL_COND_V(is_normalized() == false, Quat(0, 0, 0, 0));
+#endif
 	return Quat(-x, -y, -z, w);
 }
 
 Quat Quat::slerp(const Quat &q, const real_t &t) const {
-
+#ifdef MATH_CHECKS
+	ERR_FAIL_COND_V(is_normalized() == false, Quat(0, 0, 0, 0));
+	ERR_FAIL_COND_V(q.is_normalized() == false, Quat(0, 0, 0, 0));
```

Comment:
@mysticfall This line appears to be the line causing your issue in release_debug (but not in debug strangely, due to a compiler bug maybe?); I'm looking into it now.

Reply:
or more likely due to -ffast-math in release_debug